#goal definition
string move_type #could be: POINTCAP ; CAP
float64 x_des
float64 y_des
float64 theta_des
float64 x_cpoint
float64 y_cpoint
float64 theta_cpoint
bool reverse
bool passe
---
#result definition
float64 x_end
float64 y_end
float64 theta_end
---
#feedback
float64 x_current
float64 y_current
float64 theta_current
float64 lin_vel
float64 ang_vel

